src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -11,8 +11,8 @@ redirects:
   - /docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-usage-metrics-performance
 ---
 
-import version-trends-time-picker from 'images/version-trends-time-picker.png'
-import version-trends-ui from 'images/version-trends-ui.png'
+import versiontrendstimepicker from 'images/version-trends-time-picker.png'
+import versiontrendsui from 'images/version-trends-ui.png'
```

Comment:
You need to update "version-trends-ui" to "versiontrendsui" in:

  <img
     title="Mobile version trends UI"
     alt="A screenshot of the version trends UI that shows several app versions and the performance differences as percentages between them"
     src={version-trends-ui}
   />
   
   and then then the same for versiontrendstimepicker.